### PR TITLE
PT-323 | Enrolment table

### PR DESCRIPTION
**Pull request by jorilindell**
### Description
Enrolment table without actions dropdown

### Closes
PT-323

### Screenshot
Screenshot 2020-07-09 at 19 44 51

### Additional info
Can be tested on http://localhost:3000/en/events/palvelutarjotin:afzii4ixmu/occurrences/T2NjdXJyZW5jZU5vZGU6Nw==

### CODE REVIEW

File: src/domain/occurrence/enrolmentTable/additionalInfo/AdditionalInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    enrolment.person?.language || enrolment.studyGroup.person.language;
+  const studyGroupPersonInfo = [
+    enrolment.studyGroup.person.phoneNumber,
+    enrolment.studyGroup.person.emailAddress,
+  ].filter((e) => e);
+  const personInfo = [
+    enrolment.person?.phoneNumber,
+    enrolment.person?.emailAddress,
+  ].filter((e) => e);
+  const extraNeeds = enrolment.studyGroup.extraNeeds;
```
Comment (Sartonon) [inline src/domain/occurrence/enrolmentTable/additionalInfo/AdditionalInfo.tsx:25]: Would something like getEnrolmentField be a good fit here?

File: src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      accessor: (row: EnrolmentFieldsFragment) => row,
+      Cell: ({
+        row,
+      }: {
+        value: EnrolmentFieldsFragment;
+        row: {
+          getToggleRowExpandedProps: () => void;
+          isExpanded: boolean;
+        };
+      }) => {
```
Comment (Sartonon) [inline src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx:128]: I know that this was taken from my code :) but what do you think, should we make a new type to make this more readable?

Comment (jorilindell) [reply src/domain/occurrence/enrolmentTable/EnrolmentTable.tsx]: Tried to make more generic type for the column cell

File: src/domain/occurrence/enrolmentTable/additionalInfo/AdditionalInfo.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/occurrence/enrolmentTable/additionalInfo/AdditionalInfo.tsx]: Added helper function

**Pull request merged**